app/views/content/how-to-apply-for-teacher-training/when-to-apply-for-teacher-training.md

```diff
@@ -0,0 +1,60 @@
+---
+title: When to apply for teacher training
+description: |-
+  Find out when you can apply for postgraduate teacher training courses, deadlines for submitting applications and how the interview process works.
+related_content:
+    
+navigation: 30.45
```

Comment:
Should this appear earlier on the category page - feels like it is something you need to know early on?

Do we need any subcategories now there are lots of nav cards or is it alright as is?

Reply:
That's a good point, it would technically come first if we were going by order of process but is it the most important piece of info? Should it be 4th (before interviews)?